pages/common/rnm.md

```diff
@@ -0,0 +1,21 @@
+# rnm
+
+> Batch renaming tool.  
```

Comment:
Added two spaces at the end. Don't really know what else to do.

Reply:
In response of: https://github.com/tldr-pages/tldr/pull/7603#issuecomment-1003717302

Reply:
The problem is the empty line between this line and the more information line. Plus, there is no need to put trailing spaces in any end of lines.